### PR TITLE
feat: adding schema type (implicit or explicit) to bucket page

### LINKED ISSUE #1578: Epic: add metadata to Buckets page
### Target Customer / Background data / Context / Market Size
the target user for this feature is someone who is looking for storage information about their instance

### Goals
The goal of this epic is to incorporate the schema type information on a bucket

### Acceptance Crit
- as a user, i can see if a bucket is implicit or explicit schema type

### Notes
 - If no schema type is returned in the API, it should be assumed to be implicit schema.
 
### Release Plan
TBD

### Team
 - Anchor / Dev Champion @jrenee42 
 - Product Manager - @russorat 
 - Design - @daviddkkim

### PR DESCRIPTION
**Pull request by jrenee42**
* added unit tests
 * if the data is not there, then  the  'implicit' type is the default

Closes #1578

### CODE REVIEW

File: src/buckets/components/BucketCardMeta.test.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    readableRetention: '30 days',
+    name: 'testBucket',
+    retentionRules: [],
+    labels: [],
+  }
+  return renderWithRedux(<BucketCardMeta bucket={bucket} />)
+}
+
+describe(
+  'bucket meta data card, ' + 'testing that the schema type shows up properly',
```
Comment (hoorayimhelping) [inline src/buckets/components/BucketCardMeta.test.tsx:25]: is this string concatenation supposed to be here?

**Review by hoorayimhelping — APPROVED** [review]

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 48 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2265)

File: src/buckets/components/BucketCardMeta.test.tsx (hunk 1 above)
Comment (jrenee42) [reply src/buckets/components/BucketCardMeta.test.tsx:25]: pretty-quick added that.

Comment (jrenee42) [reply src/buckets/components/BucketCardMeta.test.tsx]: ...i fixed the whitespace.  after the build passes will merge up.

**Pull request merged**